config.json

```diff
@@ -0,0 +1,541 @@
+{
+  "settings": {
+    "CloudDropAccessToken": {
+      "description": "Azure access token",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CloudDropAccountName": {
+      "description": "Azure account name.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
+    "BuildNumberMajor": {
+      "description": "Product build major number.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
+    "BuildNumberMinor": {
+      "description": "Product build minor number.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
+    "MsBuildParameters": {
+      "description": "MsBuild building options.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "/nologo /verbosity:minimal /clp:Summary"
+    },
+    "MsBuildFileLogging": {
+      "description": "MsBuild logging options.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "/flp:v=normal"
+    },
+    "MsBuildEventLogging": {
+      "description": "MsBuild logging options.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "/l:BinClashLogger,Tools/net45/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
+    },
+    "FilterToOSGroup": {
+      "description": "Specifies the OSGroup.",
+      "valueType": "property",
+      "values": [ "Windows_NT", "Linux", "Unix", "OSX" ],
+      "defaultValue": "${OSName}"
+    },
+    "RestoreNETCorePlatforms": {
+      "description": "MsBuild target that restores the NETCore packages.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "RestoreDuringBuild": {
+      "description": "Enables/disables package restore.",
+      "valueType": "property",
+      "values": [ true, false ],
+      "defaultValue": true
+    },
+    "Project": {
+      "description": "Project where the commands are going to be applied.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "__BuildType": {
```

Comment:
In this context, what is the difference between the Settings that start with `__` and the ones that don't? Is it a convention we want to maintain?